experiment/nursery/BUILD.bazel

```diff
@@ -38,6 +38,7 @@ filegroup(
 
 go_image(
     name = "image",
+    base = "@alpine-base//image",
```

Comment:
distroless doesn't have any utilities, I've spun up alpine and it has `mount` etc.